src/app/tests/suites/certification/Test_TC_PCC_2_2.yaml

```diff
@@ -50,13 +58,21 @@ tests:
       arguments:
           value: 2
 
-    - label: "Reads the attribute: EffectiveOperationMode"
-      disabled: true
-      command: "readAttribute"
-      attribute: "EffectiveOperationMode"
+    - label:
+          "Read EffectiveOperationMode attribute from the DUT and Verify that
+          the DUT response indicates EffectiveOperationMode attribute has the
+          value 2"
+      verification: |
+          ./chip-tool pumpconfigurationandcontrol read effective-operation-mode 1 1
+      cluster: "LogCommands"
+      command: "UserPrompt"
```

Comment:
Again, why is this UserPrompt?

What is the actual goal of this PR?  Why are we making these changes?